plugins/woocommerce/client/admin/client/settings-payments/index.tsx

```diff
@@ -395,7 +395,13 @@ export const SettingsPaymentsWooPaymentsWrapper = () => {
 				title={ __( 'Settings', 'woocommerce' ) }
 				context={ 'wc_settings_payments__woopayments' }
 			/>
-			<Suspense fallback={ <div>{ __( 'Loading WooPayments settings...', 'woocommerce' ) }</div> }>
+			<Suspense
+				fallback={
+					<div>
+						{ __( 'Loading WooPayments settings…', 'woocommerce' ) }
```

Comment:
We should not make "WooPayments" translatable since it is a brand name. Best to replace it with a placeholder in the string and use `sprintf` to replace it.

Reply:
Thanks, I agree that plugin names should not be translatable. Updated!